feat: add FROM_DAYS and update UNIX_DATE function (#7742)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/scalar-functions.md

Since: 0.6.0

```sql
UNIX_DATE([date])
```

> If `UNIX_DATE` is called with the date parameter, the function returns the DATE

Reviewer: ```suggestion
If `UNIX_DATE` is called with the *date* parameter, the function returns the DATE
```

value as an INTEGER value representing the number of days since `1970-01-01`.

> If the `date` parameter is not provided, it returns an integer representing days since `1970-01-01`.

Reviewer: ```suggestion
If the `date` parameter is not provided, `UNIX_DATE` returns an integer representing the number of days since `1970-01-01`.
```

Converts a BIGINT millisecond timestamp value into a TIMESTAMP value.

### `FROM_DAYS`

```sql
FROM_DAYS(days)
```

> Converts an INT number of days since epoch to a DATE value.

Reviewer: ```suggestion
Converts the specified number of days since the Unix epoch to a DATE value. 
```
